internal/host/hostcommands/install_cmd.go

```diff
@@ -365,11 +414,6 @@ func appends390xArgs(inventory *models.Inventory, installerArgs []string, log lo
 }
 
 func appendDHCPArgs(cluster *common.Cluster, host *models.Host, inventory *models.Inventory, installerArgs []string, log logrus.FieldLogger) ([]string, error) {
-
-	if hasUserConfiguredIP(installerArgs) {
-		return installerArgs, nil
-	}
```

Comment:
why is this removed? if there is an IP we should exit

Reply:
it's not removed, this check has moved to L.295, to ensure we don't add any `ip=...` kargs if the user has already defined some.